roachtest: sqlsmith/setup=rand-tables/setting=no-ddl failed

SHA: https://github.com/cockroachdb/cockroach/commits/3b0662a83e1226dd4df2e82badacfcfd7440179f

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1575698&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191106-1575698/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:118,sqlsmith.go:129,test_runner.go:697: ping: dial tcp 34.70.215.160:26257: connect: connection refused
		previous sql:
		INSERT
		INTO
			defaultdb.public.table2 AS tab_14303 (col0, col2, col3, col4, col5, col6, col7, col9)
		WITH
			with_4730 (col_34096, col_34097)
				AS (
					SELECT
						*
					FROM
						(
							VALUES
								(
									ARRAY[(-32768):::INT8],
									ARRAY[2624302975:::OID,66313090:::OID,4051671414:::OID,1524926352:::OID]
								),
								(
									ARRAY[25831:::INT8,19000:::INT8,(-26706):::INT8,(-923):::INT8,16924:::INT8,7623:::INT8],
									ARRAY[576955916:::OID,3719283918:::OID,2623941004:::OID]
								),
								(
									ARRAY[(-32768):::INT8,19176:::INT8,(-32768):::INT8,4628:::INT8,13816:::INT8],
									(
										WITH
											with_4729 (col_34093)
												AS (
													SELECT
														*
													FROM
														(
															VALUES
																((-9050814631601846520):::INT8),
																(NULL),
																(1520710204191204877:::INT8)
														)
															AS tab_14304 (col_34093)
												)
										SELECT
											ARRAY[1831852727:::OID,3409238126:::OID] AS col_34094
										FROM
											with_4729
										LIMIT
											1:::INT8
									)
								),
								(
									(
										SELECT
											ARRAY[]:::INT2[] AS col_34095
										FROM
											defaultdb.public.table2 AS tab_14305
											JOIN defaultdb.public.table1 AS tab_14306 ON (tab_14305.col5) = (tab_14306.col3)
										LIMIT
											1:::INT8
									),
									ARRAY[2959217651:::OID]
								)
						)
							AS tab_14307 (col_34096, col_34097)
				),
			with_4731 (col_34098)
				AS (
					SELECT
						*
					FROM
						(VALUES (NULL), (0.37973940463801925:::FLOAT8), (NULL), (0.4978006121978615:::FLOAT8))
							AS tab_14308 (col_34098)
				)
		SELECT
			NULL AS col_34099,
			e'\x00':::STRING AS col_34100,
			'X':::STRING AS col_34101,
			stddev(with_4731.col_34098::FLOAT8) OVER (PARTITION BY with_4731.col_34098)::FLOAT8 AS col_34102,
			B'1101010000011110001110000011110' AS col_34103,
			e'\x06L\x13K':::STRING:::NAME AS col_34104,
			current_timestamp():::TIMESTAMP::TIMESTAMP AS col_34105,
			e':ZI9JoU\x1a|':::STRING AS col_34106
		FROM
			with_4731
		WHERE
			isnan(5.946347845456226986E+21:::DECIMAL::DECIMAL)::BOOL
		GROUP BY
			with_4731.col_34098;

```

SHA: https://github.com/cockroachdb/cockroach/commits/e206eec2aaac3620d1d27873b830bc5f781c94aa

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1580926&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191108-1580926/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:132,sqlsmith.go:159,test_runner.go:697: error: pq: internal error: uncaught error: overflow during Encode
		stmt:
		WITH
			with_803 (col_5860)
				AS (
					SELECT
						*
					FROM
						(VALUES (('3000-01-01 00:00:00+00:00':::TIMESTAMPTZ, ARRAY[]:::TIME[]))) AS tab_2187 (col_5860)
				)
		SELECT
			tab_2189.col14 AS col_5861,
			tab_2189.col2 AS col_5862,
			tab_2188.col11 AS col_5863,
			'28 years 9 mons 144 days 05:08:55.446546':::INTERVAL AS col_5864,
			'25.117.56.64/14':::INET AS col_5865,
			tab_2188.col2 AS col_5866,
			(NULL, ARRAY[]:::TIME[]) AS col_5867,
			(-1.1362269137015755):::FLOAT8 AS col_5868,
			'17ba166e-dcdc-4c5d-87f0-5dcdc35687e1':::UUID AS col_5869,
			(tab_2189.col6::DECIMAL * '43 years 4 mons 254 days 02:15:58.021547':::INTERVAL::INTERVAL)::INTERVAL AS col_5870,
			tab_2189.col14 AS col_5871,
			'Z1k':::STRING AS col_5872
		FROM
			with_803, defaultdb.public.table1 AS tab_2188, defaultdb.public.table1 AS tab_2189
		ORDER BY
			tab_2189.col10
		LIMIT
			53:::INT8;

```

SHA: https://github.com/cockroachdb/cockroach/commits/33b96613ae532b25a1b6b716453bece9b60ba2d6

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1583742&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191109-1583742/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:132,sqlsmith.go:159,test_runner.go:697: error: pq: internal error: crdb_internal.encode_key(): expected tuple argument for row_tuple, found 2646845678
		stmt:
		INSERT
		INTO
			defaultdb.public.table1 AS tab_3034
				(col18, col1, col2, col4, col5, col6, col7, col8, col9, col10, col11, col12, col15, col16, col0)
		SELECT
			e'\x07V @A5':::STRING AS col_5609,
			(COALESCE('9bcb:52b:da55:5aff:c53f:7552:9e5:e6e8/20':::INET, 'ef06:5fd5:8955:2be9:738d:f0d:c7fb:565b/51':::INET)::INET - NULL::INT8)::INET
				AS col_5610,
			(-0.9241251349449158):::FLOAT8 AS col_5611,
			'1990-02-19':::DATE AS col_5612,
			'12:02:24.866489':::TIME AS col_5613,
			'76 years 738 days 04:05:33.120148':::INTERVAL AS col_5614,
			'1970-01-02 22:43:50.000277+00:00':::TIMESTAMPTZ AS col_5615,
			NULL AS col_5616,
			crdb_internal.encode_key((-7962554072520099111):::INT8::INT8, (-4528229436147324075):::INT8::INT8, 2646845678:::OID)::BYTES
				AS col_5617,
			'1970-01-11 09:43:55.000752+00:00':::TIMESTAMPTZ AS col_5618,
			'117.6.99.23/8':::INET AS col_5619,
			1.0:::FLOAT8 AS col_5620,
			e'Q\x04\x03':::STRING AS col_5621,
			3256098075405848947:::INT8 AS col_5622,
			2929717503:::OID AS col_5623;

```

SHA: https://github.com/cockroachdb/cockroach/commits/db252551659b8c404be584e39f2af642d01caee3

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1602108&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191121-1602108/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:155,sqlsmith.go:177,test_runner.go:697: error: pq: communication error: rpc error: code = Canceled desc = context canceled
		stmt:
		WITH
			with_2501 (col_16128)
				AS (SELECT '38bb1e37-5850-4bfc-8ffd-9d8ba7c7456f':::UUID AS col_16128 FROM defaultdb.public.table2 AS tab_7723)
		SELECT
			cluster_logical_timestamp()::DECIMAL AS col_16129
		FROM
			defaultdb.public.table5 AS tab_7724
		ORDER BY
			tab_7724.col1 DESC, tab_7724.col1 DESC, tab_7724.col0 DESC
		LIMIT
			91:::INT8;

```

SHA: https://github.com/cockroachdb/cockroach/commits/9012e6449e63baca6b59c00d3e350bbc3ab0dd3b

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1611118&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191126-1611118/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:150,sqlsmith.go:177,test_runner.go:697: error: pq: internal error: uncaught error: overflow during Encode
		stmt:
		SELECT
			(COALESCE(NULL, 7.238163171805024011E+21:::DECIMAL)::DECIMAL * tab_7210.col1::INTERVAL)::INTERVAL AS col_14708,
			tab_7210.col2 AS col_14709,
			tab_7210.col0 AS col_14710,
			tab_7210.col6 AS col_14711,
			tab_7210.col4 AS col_14712
		FROM
			defaultdb.public.table2 AS tab_7210
		ORDER BY
			tab_7210.col4 ASC, tab_7210.col6, tab_7210.col4 DESC, tab_7210.col6 ASC, tab_7210.col2 ASC, tab_7210.col5;

```

SHA: https://github.com/cockroachdb/cockroach/commits/ed717cbaf741e3a32c76db25b16a59dc2a8221d7

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1624103&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191204-1624103/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:162,sqlsmith.go:190,test_runner.go:697: error: pq: internal error: unsupported type bit
		stmt:
		SELECT
			3300157557:::OID AS col_4407, tab_2392.col3 AS col_4408
		FROM
			defaultdb.public.table4 AS tab_2391 JOIN defaultdb.public.table1 AS tab_2392 ON (tab_2391.col2) = (tab_2392.col3)
		LIMIT
			65:::INT8;

```

SHA: https://github.com/cockroachdb/cockroach/commits/e1c1a9ae765730782f3adc2298a76d6a07ad055d

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1626298&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191205-1626298/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:162,sqlsmith.go:190,test_runner.go:697: error: pq: internal error: unsupported type timestamptz
		stmt:
		SELECT
			tab_1529.col9 AS col_3410
		FROM
			defaultdb.public.table3 AS tab_1529
		ORDER BY
			tab_1529.col9 ASC, tab_1529.col5 DESC
		LIMIT
			12:::INT8;

```

SHA: https://github.com/cockroachdb/cockroach/commits/28f216e1bd53da872a759a98779144c7f70f33a3

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1629573&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191206-1629573/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:163,sqlsmith.go:191,test_runner.go:697: error: pq: internal error: unsupported type interval
		stmt:
		WITH
			with_848 (col_5864)
				AS (
					SELECT
						*
					FROM
						(
							VALUES
								(3634497112:::OID),
								(3742876867:::OID),
								(COALESCE(4156998938:::OID, NULL)),
								(1208317048:::OID),
								(NULL),
								(149311672:::OID)
						)
							AS tab_2484 (col_5864)
				),
			with_849 (col_5865) AS (SELECT * FROM (VALUES (2462435191:::OID)) AS tab_2485 (col_5865))
		SELECT
			count(*) AS col_5866
		FROM
			defaultdb.public.table3 AS tab_2486
		GROUP BY
			tab_2486.col7
		ORDER BY
			tab_2486.col7 DESC, tab_2486.col7;

```

SHA: https://github.com/cockroachdb/cockroach/commits/9d697080f227324220c09d1b63da7bbf969133ac

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1633532&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191209-1633532/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:163,sqlsmith.go:191,test_runner.go:697: error: pq: internal error: unsupported type timestamptz
		stmt:
		SELECT count(*) AS col_3109 FROM defaultdb.public.table3 AS tab_1488 GROUP BY tab_1488.col1, tab_1488.col8;

```

[(roachtest).sqlsmith/setup=rand-tables/setting=no-ddl failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634692&tab=buildLog) on [provisional_201912092102_v19.2.2@c85d435c2caa7a3066b1affdcc34b7ed6a6f3653](https://github.com/cockroachdb/cockroach/commits/c85d435c2caa7a3066b1affdcc34b7ed6a6f3653):

```
The test failed on branch=provisional_201912092102_v19.2.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191209-1634692/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:163,sqlsmith.go:191,test_runner.go:700: error: pq: internal error: unsupported type time
		stmt:
		SELECT
			e'\x17@:2':::STRING AS col_8402
		FROM
			defaultdb.public.table2 AS tab_4127
		ORDER BY
			tab_4127.col1 ASC, tab_4127.col10 ASC, tab_4127.col9 DESC, tab_4127.col2 DESC
		LIMIT
			85:::INT8;

```

<details><summary>details</summary>
<p>


Artifacts: [/sqlsmith/setup=rand-tables/setting=no-ddl](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634692&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl)
```
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(roachtest).sqlsmith/setup=rand-tables/setting=no-ddl failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634867&tab=buildLog) on [master@734e357412dadafcd6084b8eab8e251e44e86b4a](https://github.com/cockroachdb/cockroach/commits/734e357412dadafcd6084b8eab8e251e44e86b4a):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191210-1634867/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:163,sqlsmith.go:191,test_runner.go:700: error: pq: internal error: unsupported type bit
		stmt:
		SELECT
			count(*) AS col_19040
		FROM
			defaultdb.public.table1 AS tab_9962
		WHERE
			true
		GROUP BY
			tab_9962.col2
		ORDER BY
			tab_9962.col2 ASC
		LIMIT
			31:::INT8;

```

<details><summary>details</summary>
<p>


Artifacts: [/sqlsmith/setup=rand-tables/setting=no-ddl](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634867&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl)
```
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(roachtest).sqlsmith/setup=rand-tables/setting=no-ddl failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1649932&tab=buildLog) on [master@2f1e342c386973c35246bef68c177fcd0b8b609b](https://github.com/cockroachdb/cockroach/commits/2f1e342c386973c35246bef68c177fcd0b8b609b):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191218-1649932/sqlsmith/setup=rand-tables/setting=no-ddl/run_1
	sqlsmith.go:189,sqlsmith.go:217,test_runner.go:700: error: pq: internal error: the first bucket should have NumRange=0
		stmt:
		SELECT
			tab_2215.col9 AS col_5598,
			tab_2215.col2 AS col_5599,
			e'n\\U\x04':::STRING AS col_5600,
			e'T>)M[\x0b\x1d':::STRING AS col_5601,
			'1970-01-07 12:33:07.000458+00:00':::TIMESTAMP AS col_5602,
			'1.86.172.14/16':::INET AS col_5603,
			tab_2215.col9 AS col_5604,
			tab_2215.col5 AS col_5605,
			tab_2215.col10 AS col_5606,
			'1991-12-29':::DATE AS col_5607
		FROM
			(
				SELECT
					e'fzh\x04j':::STRING AS col_5596
				FROM
					defaultdb.public.table4@primary AS tab_2212, defaultdb.public.table5@[0] AS tab_2213
				LIMIT
					44:::INT8
			)
				AS tab_2214 (col_5597)
			JOIN defaultdb.public.table5@table5_col4_col5_idx AS tab_2215 ON
					(tab_2214.col_5597) = (tab_2215.col5) AND (tab_2214.col_5597) = (tab_2215.col1)
		WHERE
			false
		LIMIT
			91:::INT8;
```

<details><summary>Repro</summary><p>


Artifacts: [/sqlsmith/setup=rand-tables/setting=no-ddl](https://teamcity.cockroachdb.com/viewLog.html?buildId=1649932&tab=artifacts#/sqlsmith/setup=rand-tables/setting=no-ddl)
```
make stressrace TESTS=sqlsmith/setup=rand-tables/setting=no-ddl PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/sql/mutations/mutations.go
+++ b/pkg/sql/mutations/mutations.go
@@ -157,7 +157,11 @@ func statisticsMutator(
 				} else {
 					seen[es] = true
 				}
-				numRange := randNonNegInt(rng)
+				var numRange int64
+				// The first bucket should have NumRange=0.
+				if i > 0 {
+					numRange = randNonNegInt(rng)
+				}
 				var distinctRange float64
 				// distinctRange should be <= numRange.
 				switch rng.Intn(3) {

